test-harness/build.gradle

```diff
@@ -26,3 +26,23 @@ task sourcesJar(type: Jar) {
 artifacts {
   archives jar, javadocJar, sourcesJar
 }
+
+uploadArchives {
+  repositories {
+    mavenDeployer {
+      pom.project {
+        name 'Test Harness for SpotBugs Plugin'
+        description 'A test harness library for SpotBugs plugin developers'
+
+        licenses {
+          // This project contains copied source files from SonarSource and find-sec-bugs,
+          // which is licensed under LGPL 3.0, so we should use v3.0 instead of 2.1.
```

Comment:
find-sec-bugs is not just me. But the code use was the small test utility (hamcrest matchers mostly) which I was only the only contributor. I gave authorization to @jsotuyod to transfer the code to spotbugs.

![2017-02-20 11_47_33-improve infrastructure for better integration tests by jsotuyod pull request](https://cloud.githubusercontent.com/assets/593130/23134336/71f79f46-f762-11e6-8e7d-1ddac1093e99.png)

-----

The code use from SonarSource is actually not under LGPL3.0. It is a SmapParser that was reused. Its main license is just to keep a header mentioning the original author.
https://github.com/spotbugs/spotbugs/blob/1aa089627fb3639634d7cfa4dbfeaf063e9917cd/findbugs/src/junit/org/sonar/plugins/findbugs/resource/SmapParser.java#L39

Original source: http://stackoverflow.com/a/11299757

This segment could be removed : https://github.com/spotbugs/spotbugs/blob/1aa089627fb3639634d7cfa4dbfeaf063e9917cd/findbugs/src/junit/org/sonar/plugins/findbugs/resource/SmapParser.java#L2

The few changes were made allow the support additional JSP compilers. I do not recommend using the original Stackoverflow source.

In conclusion, the dependency to Sonar-FindBugs license should not exist. Only the reference to Michael Schierl in the source.

----

It may not be 100% legally accurate. But since I produce the test utility and the small fixes to the SMAPParser, I can assure I won't sue the project. ;)